src/nvim/api/vim.c

```diff
@@ -2882,3 +2883,85 @@ Array nvim_get_mark(String name, Error *err)
   return rv;
 }
 
+///
+/// Set matches and start insert completion: see |complete()|.
```

Comment:
For reference, what are the reasons for adding this API function instead of augmenting the existing `complete()`?